PathKit/Sources/PathKit/Line.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright © 2019 JABT Labs. All rights reserved.
+//
+// This file is part of PathKit. The full PathKit copyright notice,
+// including terms governing use, modification, and redistribution, is
+// contained in the file LICENSE at the root of the source code distribution
+// tree.
+
+import Foundation
+
+/// A line in 2-dimensional space represented in its normal form.
+public struct Line {
+    /// The inclination of the line in radians.
+    public var 𝜭: Angle
+
+    /// The length of the normal segment.
+    /// The normal segment joins the origin with the closest point on the line to the origin.
+    public var p: Double
+
+    /// Determines whether `point` is on the line.
+    public func contains(_ point: Point) -> Bool {
```

Comment:
let's add a tolerance parameter as we discussed.

Reply:
Yeah for sure, in the next PR I am going to introduce the tolerance parameter and apply it through out all the structs, Angle, Vector, Line Segment, Line etc.
And I'll create some unit tests and use that to test the CI.

Reply:
will do that tonight